monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -240,6 +254,23 @@ enable_pod_token_automount $MON_NS deployment v4m-operator
 log_info "Deploying ServiceMonitors and Prometheus rules"
 log_verbose "Deploying cluster ServiceMonitors"
 
+if [ "$TRACING_ENABLE" == "true" ]; then
+  TEMPO_CHART_VERSION=${TEMPO_CHART_VERSION:-1.5.0}
+  log_info "Tracing enabled..."
+  # # Add the grafana helm chart repo
+  helmRepoAdd grafana https://grafana.github.io/helm-charts
+  helm repo update
+
+  log_info "Installing tempo"
+  helm upgrade --install v4m-tempo \
+    -n "$MON_NS" \
+    -f "$TEMPO_USER_YAML" \
+    --set serviceMonitor.enabled=true \
+    --set searchEnabled=true \
+    --version "$TEMPO_CHART_VERSION" \
```

Comment:
Future Enhancement:  let's put our "default" values in a yaml file.  That would be consistent with how we've handled the other Helm charts and will allow us to adjust settings without having to edit the script.